ambry-clustermap/src/test/java/com.github.ambry.clustermap/TestUtils.java

```diff
@@ -345,6 +350,51 @@ public static JSONObject getJsonPartitionLayout(String clusterName, long version
     return jsonObject;
   }
 
+  /**
+   * Verify that the partitions in the list are on unique racks for each datacenter.
+   *
+   * @param allocatedPartitions the list of partitions to check
+   */
+  public static void checkRackUsage(List<PartitionId> allocatedPartitions) {
```

Comment:
would this function be useful as a general utility function? What other test might potentially have a use for this function?